packages/@o3r/eslint-config-otter/package.json

```
@@ -50,15 +50,15 @@
     "build:builders": "tsc -b tsconfig.builders.json --pretty && yarn generate-cjs-manifest"
   },
   "peerDependencies": {
-    "@angular-devkit/schematics": "^18.2.0",
-    "@angular-eslint/builder": "^18.4.0",
-    "@angular-eslint/eslint-plugin": "^18.4.0",
-    "@angular-eslint/eslint-plugin-template": "^18.4.0",
-    "@angular-eslint/template-parser": "^18.4.0",
-    "@angular/compiler": "^18.2.0",
+    "@angular-devkit/schematics": "^19.0.0",
+    "@angular-eslint/builder": "^18.4.0 || ^19.0.0",
+    "@angular-eslint/eslint-plugin": "^18.4.0 || ^19.0.0",
+    "@angular-eslint/eslint-plugin-template": "^18.4.0 || ^19.0.0",
+    "@angular-eslint/template-parser": "^19.0.2",
```

```suggestion
    "@angular-eslint/template-parser": "^18.4.0 || ^19.0.2",
```
shouldn't it be the same for all angular peer deps?

---

I don't think so. I didn't encounter any issues with legacy lint config and Angular 19.